lua/entities/gmod_wire_expression2/core/console.lua

```diff
@@ -35,18 +35,19 @@ local function tokenizeAndGetCommands(str)
 	return commands
 end
 
----@param cmd_var "cmd"|"var"
----@return boolean override # Whether the user set their whitelist to "" (allow everything)
----@return table whitelist # Whitelist for specific commands otherwise.
-local function getWhitelist(ply, cmd_var)
-	local whitelist = (ply:GetInfo("wire_expression2_con" .. cmd_var .. "_whitelist") or ""):Trim()
-	if whitelist == "" then return true end
+---@param cvar "wire_expression2_concmd_whitelist"|"wire_expression2_convar_whitelist"
+---@return table whitelist # Whitelist for specific commands, if empty, disregard whitelist and allow everything
+local function getWhitelist(ply, cvar)
+	local whitelist = (ply:GetInfo(cvar) or ""):Trim()
 
 	local whitelistTbl = {}
+	if whitelist == "" then return whitelistTbl end
```

Comment:
Probably don't need this since the for loop won't do anything if the string is empty